nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -325,7 +325,7 @@ private static String addRecordsToFile(Table table, String filename) throws IOEx
     String fileLocation = table.location().replace("file:", "") +
         String.format("/data/%s.avro", filename);
     try (FileAppender<GenericData.Record> writer = Avro.write(Files.localOutput(fileLocation))
-        .schema(schema)
+        .schema(SCHEMA)
```

Comment:
In the future, let's separate style changes from behavior or features. It's okay now because this is unreleased so no one is submitting patches or cherry-picking into a branch. But in the future we prefer to avoid commit conflicts by keeping PR changes small and focused.

Reply:
ack